pkg/util/trace.go

```diff
@@ -0,0 +1,27 @@
+package util
+
+import (
+	"errors"
+
```

Comment:
Looks like a newline got into the imports! Can this please be removed real quick before a merge?